.env.template

```diff
@@ -12,4 +12,19 @@ OWN_IP="127.0.0.1"
 EXTRA_ALLOWED_HOSTS = "*"
 
 #Optional Setting; For regex matching the game URLs for the bot to recognise it as valid. Used in bot,py
-GAME_URL = "si.bitcrafter.net"
\ No newline at end of file
+GAME_URL = "si.bitcrafter.net"
+
+# DJango server settings
+DJANGO_SUPERUSER_EMAIL="foo@bar.com"  # email for admin user
+DJANGO_SUPERUSER_USERNAME="root"      # username for admin user
+DJANGO_SUPERUSER_PASSWORD="hunter2"   # password for admin user
```

Comment:
are these three `DJANGO_SUPERUSER_` items being used in any way? I can't find mentions of them in the code. Does Django know to look for them?